code/modules/fish/fish_tank.dm

```diff
@@ -804,7 +804,7 @@
 	if(check_tank())
 		for(var/fish in attached_tank.fish_list)
 			if(fish == "electric eel")
-				power += ARBITRARILY_LARGE_NUMBER * multiplier //10000
+				power += 10000 * multiplier //10000
```

Comment:
why is their still a //10000 here

Reply:
This is an awful comment that I actually looked at when making this one line change and actively decided to ignore. Let it be a curse to future coders who step here. A mystery: why is this comment here. The number is obvious. Why state it again in a comment and bloat things. They might look at the history and find something great, struck down by the whims of an oldcoder.

Reply:
how about this - put a new define at the top of the file called EELPOWER or something, set that to 10000
then keep the comment on that line if you really must

i think its better to name the number rather than leave it as a magic number * a variable called `multiplier`. it also makes future changes to the value a lot easier to understand rather than a random number in the middle of a function